hooks/pre/10-reset_feature.rb

```diff
@@ -57,4 +63,11 @@ def reset_pulp
   Kafo::Helpers.execute(commands)
 end
 
-reset if app_value(:reset) && !app_value(:noop)
+if app_value(:reset) && !app_value(:noop)
+  response = ask('Are you sure you want to continue? [y/n]')
```

Comment:
Mind putting a bit more information in here for context for the user? Perhaps warn them what reset will do and ask if they wish to continue?